docs/v1/user-story.md

```diff
@@ -0,0 +1,124 @@
+# A Year In The Life Of A Kubernetes Service Developer
+
+## 0. Today
+I'ma developer in an enterprise environment.  For some fun in my spare time, I’m building a node.js chat application for me 
+and a couple of coworkers to goof around with.  It’s really simple.  It runs on one container.  To get going, I simply 
+download a node.js container onto minikube on my laptop and started coding.  I want some co-workers to be able to find the 
+chat app to connect.  So, I register it as a service in service in the kubernetes service catalog.  It keeps track of the IP 
```

Comment:
Moving the app from laptop to prod, across networks, etc. is fairly unrealistic, but it does raise the issue of how to invalidate binding information.